### PR TITLE
Enable MacOS Nightly Builds

### PR DESCRIPTION
**Pull request by DavidMansolino**
Remaning tasks:
  - [x] Fix installation image.
  - [x] Fix java version.

### CODE REVIEW

File: src/packaging/webots_distro.c [hunk 1]
```diff
@@ -942,7 +943,9 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "' | osascript\n");
       fprintf(fd, "chmod -Rf go-w \"/Volumes/%s\"\n", application_name);
       fprintf(fd, "sync\n");
+      fprintf(fd, "sleep 5\n");
       fprintf(fd, "sync\n");
+      fprintf(fd, "sleep 5\n");
```
Comment (DavidMansolino) [inline src/packaging/webots_distro.c]: Adding sleeps here fixes the creation of the package on travis for macOS.

I am currently testing the package creation on my fork before merging this: https://travis-ci.org/DavidMansolino/webots-test/builds/547575865

Comment (omichel) [reply src/packaging/webots_distro.c]: Sounds like sorcery... Thank you Apple! 🙈

Comment (DavidMansolino) [reply src/packaging/webots_distro.c]: It seems to work fine: https://github.com/DavidMansolino/webots-test/releases/tag/nightly_19_6_2019

**Comment by DavidMansolino** [comment]
Only remaining issue: the osascript used to set the installation image is causing a timeout, the result is that when installing the package we don't see the installation image. Not sure how to fix this, I tried to increase default timeout but it didn't change, not sure we can afford using finder on travis.

**Comment by omichel** [comment]
It may be worth checking this: https://github.com/andreyvit/create-dmg

**Comment by DavidMansolino** [comment]
> It may be worth checking this: https://github.com/andreyvit/create-dmg

In fact I am currently trying this option right now :smile: It seems indeed to simplify everything!

**Comment by DavidMansolino** [comment]
I have successfully created the package with `create-dmg`, but it doesn't help:
  - It appears they are also using Finder: https://github.com/andreyvit/create-dmg/blob/master/support/template.applescript#L2 which cause the exact same problem for creating the installation window.
  - They have also sometimes a disk detach failure (which was apparently now fixed in our home-made version of the script), they aslo have arbitrary wait/delay call: https://github.com/andreyvit/create-dmg/blob/master/support/template.applescript#L61

**Comment by DavidMansolino** [comment]
:+1: 

The result can be tested on my fork: https://github.com/DavidMansolino/webots-test/releases/tag/nightly_21_6_2019

**Comment by fabienrohrer** [comment]
👀

**Comment by DavidMansolino** [comment]
:+1:

**Comment by DavidMansolino** [comment]
Test suite passed on my fork: https://travis-ci.org/DavidMansolino/webots-test/builds/550135521?utm_source=github_status&utm_medium=notification

**Pull request merged**